scripts/custom/aica_bridge/bridge/aica_bridge.py

```diff
@@ -0,0 +1,103 @@
+from typing import Optional, Union, List
+
+import torch
+from isaaclab.assets import Articulation
+from isaaclab.scene import InteractiveScene
+
+from communication_interfaces.sockets import (
+    ZMQCombinedSocketsConfiguration,
+    ZMQContext,
+    ZMQPublisher,
+    ZMQPublisherSubscriber,
+    ZMQSocketConfiguration,
+)
+import clproto
+import state_representation as sr
+from state_representation import StateType
+from scripts.custom.aica_bridge.bridge.utils import measured_forces
+from scripts.custom.aica_bridge.bridge.config_classes import BridgeConfig
+
+class AICABridge:
+    def __init__(self, config: BridgeConfig, robot_joint_ids: List[int] | slice):
```

Comment:
Here in particular, a docstring about where to get the robot_joint_ids from would be useful (i.e. the result of `SceneEntityCfg`)